src/components/calendar/CalendarSection.js

```diff
@@ -67,7 +67,7 @@ export default function CalendarSection({ section, isPreview }) {
         classes={classes}
         leftHeaderContent={leftHeaderContent}
         rightHeaderContent={rightHeaderContent}
-        sectionName={section.name}
+        sectionName={(section.topic ==='') ? section.name : section.topic}
```

Comment:
No need to wrap that first boolean in parentheses – it also looks like linting didn't catch that weird spacing between the `===` and the `''`, so we'll have to talk about getting that set up properly.